ContactDetailsApi/V2/Boundary/Request/Validation/AddressExtendedValidator.cs

```diff
@@ -0,0 +1,64 @@
+using ContactDetailsApi.V1.Boundary.Request.Validation;
+using ContactDetailsApi.V1.Domain;
+using FluentValidation;
+using Hackney.Core.Validation;
+using AddressExtended = ContactDetailsApi.V2.Domain.AddressExtended;
+
+namespace ContactDetailsApi.V2.Boundary.Request.Validation
+{
+    public class AddressExtendedValidator : AbstractValidator<AddressExtended>
+    {
+        private const string PostCodeRegEx = @"^((([A-Za-z][0-9]{1,2})|(([A-Za-z][A-Ha-hJ-Yj-y][0-9]{1,2})|(([A-Za-z][0-9][A-Za-z])|([A-Za-z][A-Ha-hJ-Yj-y][0-9][A-Za-z]))))( )?(([0-9][A-Za-z]?[A-Za-z]?)?))$";
```

Comment:
This is a candidate for another common validator in Hackney.Core.Validation - but that's for another ticket.